code/src/components/Details.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react'
+import styled from "styled-components";
+import { useParams, useNavigate } from 'react-router-dom'
+import LoadingPage from './LoadingPage';
+
+const baseURL = "https://image.tmdb.org/t/p/";
+const imgSize = "original";
+
+const BackDrop = styled.div`
+        background-image: linear-gradient(180deg, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 100%), url(${(props) => props.url});
+        background-size: cover;
+        min-height: 100vh;
+`;
+
+export const Details = ({API_KEY, StyledButton, loading, setLoading}) => {
+    const {movieId} = useParams()
+    const [movie, setMovie] = useState({})
+    const navigate = useNavigate();
+
+    const onBackButtonClick = ()=> {
+        navigate(-1);
+    }
+
+    const API = `https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`
+
+    useEffect(() => {
+        fetch(API)
+            .then((res) => res.json())
+            .then((data) => {
+                setMovie(data)
+                setLoading(false)
+            });
+    }, [API, setLoading])

```

Comment:
We didn't do a second API call inside the individual movies, so I'm curious if it provided any extra additional information that the API call in the PopularList. Please let me know if it gave some extra info! Very curious.